react/week4/my-app/src/Context.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState, useEffect } from "react";
+
+const Context = React.createContext();
+
+function ContextProvider({ children }) {
+  const [userInput, setUserInput] = useState("");
+  const [data, setData] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  const fetchGitUsers = async (input) => {
+    try {
+      await fetch(`https://api.github.com/search/users?q=${input}`)
+        .then((response) => {
+          if (!response.ok) {
+            throw Error("Could not fetch the data ");
+          } else {
+            return response.json();
+          }
+        })
+        .then((data) => {
+          setData(data.items);
+          //console.log(data);
+          setLoading(false);
+        });
+    } catch (error) {
+      setError(error.message);
+    }
```

Comment:
Did you intend to use async/await together with `.then`? Usually, you use async/await or then/catch/finally but mixing them reads a bit weird.

Reply:
no I just forgot to take them away , because at first I wasn't wrote like that🤕

Reply:
I will fix it